README.md

```diff
@@ -73,7 +74,9 @@ Parameter | Description | Default
 | **Watcher** |
 `watcher.kubernetes.create` | If true, Kubernetes watcher will be deploy | `true`
 `watcher.kubernetes.annotations` | The annotations used in api deployment | `{}`
-`watcher.kubernetes.application.log_level` | The application log level | `info`
+`watcher.kubernetes.application.log.level` | The application log level | `info`
+`watcher.kubernetes.application.log.gelf_address` | The address for ship the log out for external system | undefined
+`watcher.kubernetes.application.log.level` | The application log level | `info`
```

Comment:
you have it twice

Reply:
Ops, my mistake. 
The line was removed.